app/models/user.rb

```diff
@@ -1,4 +1,5 @@
 class User < ActiveRecord::Base
   has_secure_password
   validates :email, presence: true, uniqueness: true
+  validates_format_of :email, :with => /\A[^@\s]+@([^@\s]+\.)+[^@\s]+\z/
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer the new style validations `validates :column, format: value` over `validates_format_of`.